frontend/src/pages/Projects/projectlist.ts

```diff
@@ -1,21 +1,40 @@
 export const projectlist = [
   {
-    projectName: "BitBot",
-    lang: "Typescript",
+    projectName: "GetIt",
+    lang: "Nuxt, Django, Javascript",
     description:
-      "A small lazy discord bot to inspire the development-bug inside me",
+      "Streamlined college e-commerce: Simplifying merch buying and selling for students.",
     createdBy: "BitByte",
-    url: "https://github.com/BitByte-TPC/BitBot",
-    image: "/webix.iiitdmj.ac.in/images/projects/bitbot.png",
+    url: "https://github.com/bsoc-bitbyte/GetIt",
+    image: "/webix.iiitdmj.ac.in/images/projects/get_it.png",
   },
   {
-    projectName: "Contest Saathi",
+    projectName: "Fend",
```

Comment:
Why does it include changes in the project list? Please create a separate PR for project list changes.

Reply:
Updated the PR. 
Incorrectly created a branch, leading to commits being tied. Will resolve the same and create a new PR for project-list changes.